<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManager</class>
 <widget class="QWidget" name="UserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1756</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>361</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="searchUser_LineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="searchUser_Btn">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>14</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="user_TableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>341</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="selectUser_Btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>810</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditUser_Btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>810</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUser_Btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>810</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newUser_Btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>810</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="TrainRecord" name="trainRecord_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>1271</width>
     <height>831</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrainRecord</class>
   <extends>QWidget</extends>
   <header>trainrecord.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
